app/components/RegionDropdown/RegionDropdown.tsx

```diff
@@ -16,14 +17,24 @@ const options: Option[] = [
 ];
 
 type RegionDropdownProps = {
-  selectedRegion: RegionOptionId;
+  selectedRegion: RegionOptionId | string | undefined;
   onChange: (id: RegionOptionId) => void;
 };
 
 export const RegionDropdown = ({
   selectedRegion,
   onChange,
 }: RegionDropdownProps) => {
+  useEffectOnce(() => {
+    const isValidSelectedRegion =
+      selectedRegion &&
+      options.map((o) => o.id as string).includes(selectedRegion);
+
+    if (!isValidSelectedRegion) {
+      onChange("auckland");
+    }
+  });
```

Comment:
Checks initial value is a valid region, if not, defaults to `auckland` - chosen it to work like this because:
- We default it to auckland in the parent
- The region select defaults to auckland being selected.

This makes it consistent 🙏 - we could add a disabled option, which could be selected by default? That would mean we wouldn't have to run this check 🤔 

![image](https://user-images.githubusercontent.com/22784983/141921898-ec81e62a-404e-4e66-b3cb-f94916b72455.png)

Reply:
Actually we can do 1 better, a hidden option will show in the 'select' but not in the options list. And only when the query param is invalid 🤔 I think I'm gonna do that 🙏